support-frontend/stories/contributionTicker.jsx

```diff
@@ -21,6 +21,7 @@ stories.add('Contribution ticker', () => (
     <ContributionTicker
       tickerJsonUrl={tickerJsonUrlInput()}
       onGoalReached={() => {}}
+      tickerType="unlimited"
```

Comment:
doesn't have to be in this PR, but we should make this configurable through the storybook UI so people can easily experiment with the two types

Reply:
Yeah we were just talking about that! But now ticker.json is sending a cors error to storybook 😢 so it's TBD